earth_enterprise/src/common/khRefCounter.h

```diff
@@ -84,7 +84,7 @@ class khRefGuard {
 
   // expose refcount function from my shared object
   inline uint32 refcount(void) const  { return ptr ? ptr->refcount() : 0; }
-
+  inline uint64 getRefGuardSize(void) const { return ptr->GetSize(); }
```

Comment:
Minor: could include the size of the pointer itself: `return sizeof(ptr) + ptr->GetSize()`. Won't make a huge difference, though.

Reply:
Was calculating this but moved the sizeof(ptr) call into this function for more clarity.